Assets/Countly/Plugins/CountlySDK/Services/UserDetailsCountlyService.cs

```diff
@@ -374,7 +367,14 @@ private void AddToCustomData(string key, object value)
             CustomDataProperties.Add(key, value);
         }
 
-        #region override Methods
-        #endregion
+        public bool ContainsCustomDataKey(string key)
```

Comment:
are those necessary or the new functions to introduce?

Reply:
class was exposing 
`internal Dictionary<string, object> CustomDataProperties { get; private set; }`
it was exposed for testing functionality. introduced methods solve the issue and also can be used without problem in the user end i.m.o.

would love @turtledreams 's opinion on them